examples/benchmark/benchmark.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
I would also consider to add "-e" flag to stop the execution in case of an error